test_opensearch/run_tests.py

```diff
@@ -113,18 +117,34 @@ def run_all(argv=None):
             ignores.extend(
                 [
                     "test_opensearch/test_server/",
+                    "test_opensearch/test_server_secured/",
```

Comment:
It seems it should be added to ignores only for the case when `not secured` -> done on line 140.

Reply:
Ah, I see, it is for `if "GITHUB_ACTION" in environ:`